examples/teleop-peer/src/main.ts

```diff
@@ -12,27 +19,40 @@ el("button").addEventListener("click", async () => {
     // hide intro
     el("section").style.display = "none";
     el("#log").style.display = "block";
-
-    // connect directly to a device (agent) via its IP.
-    // NOTE: you must set the FORMANT_AGENT_IP ENV on the agent to an IP reachable by this UI.
-    // you can use "0.0.0.0" if you would like to listen on all interfaces
-    const device = await Fleet.getPeerDevice("http://10.4.4.182:5502");
+    // Just connect to whereever this is hosted on port 5502
+    const host = window.location.origin.replace(/:[0-9]+$/, '');
```

Comment:
I think we should avoid changing the functionality of this example, as it's intended to be a standalone demo that someone runs on their local system to connect to an agent (not hosted on the agent itself).

I do think it'd be reasonable to make this change once the new offline mode feature has been released, though.